ansible/playbooks/deploy_moriod.yml

```diff
@@ -0,0 +1,118 @@
+---
+- name: Provision EC2 Instances and Configure DNS
+  hosts: localhost
+  gather_facts: no
+  collections:
+    - amazon.aws
+    - community.general
+  vars:
+    region: "eu-central-1"
```

Comment:
Please read all of these values from the Github Actions secrets:

- `subnet_id`: `${{ secrets.ITEST_AWS_SUBNET_ID }}`
- `security_group_id`: `${{ secrets.ITEST_AWS_SECURITY_GROUP_ID }}`
- `zone_id`: `${{ secrets.ITEST_AWS_ROUTE53_ZONE_ID }}`
- `ami_id`: `${{ secrets.ITEST_AWS_DEBIAN12_AMI_ID }}` (we will later also test on redhat-based distros)

In general, keep in mind that the threat model for CERT-EU is very different from your average organisation.
So in this case, we will rotate all of these so that the values in this PR become irrelevant.

Long story short: Do not put anything in the repo that can give attackers an advantage, no matter how small.

Reply:
I've used Github Actions secrets instead.